code/modules/hydroponics/plants_alien.dm

```diff
@@ -147,6 +147,19 @@ ABSTRACT_TYPE(/datum/plant/artifact)
 	endurance = 0
 	assoc_reagents = list("plasma")
 
+/datum/plant/artifact/leaker
+	name = "Leaker"
```

Comment:
overall, any reason not for this to be a mutation of the dripper or smth?

Reply:
mostly because the puker wasn't a mutation